rskj-core/src/main/java/co/rsk/peg/federation/PendingFederation.java

```diff
@@ -170,4 +167,47 @@ public int hashCode() {
         // well-defined hashCode()
         return Objects.hash(getBtcPublicKeys());
     }
+
+    public byte[] serialize(ActivationConfig.ForBlock activations) {
+        if (activations.isActive(ConsensusRule.RSKIP123)) {
+            return serializeFromMembers();
+        } else {
+            return serializeOnlyBtcKeys();
+        }
```

Comment:
```suggestion
        if (!activations.isActive(ConsensusRule.RSKIP123)) {
            return serializeOnlyBtcKeys();
        }
        return serializeFromMembers();
```
Just an idea, I like that it shows clearly what is the current behaviour

Reply:
good one